lib/browser.ts

```diff
@@ -1,3 +1,10 @@
 import AlgoliaAnalytics from "./insights";
+import { processQueue } from "./_processQueue";
 
-export default new AlgoliaAnalytics();
+const instance = new AlgoliaAnalytics({ processQueue });
```

Comment:
The argument is not used anymore.

Reply:
https://github.com/algolia/search-insights.js/pull/141/commits/b87900fabcca6e7039901ec617032799a93ce765